bevrand.multivuerse/src/router/index.js

```diff
@@ -1,8 +1,17 @@
 import Vue from 'vue'
 import Router from 'vue-router'
+import store from '../store';
 
 Vue.use(Router);
 
+function guard(to, from, next){
+    if(store.state.loggedIn) {
```

Comment:
Perhaps a linter could be added to establish more consistent use of spacing, or use of quote types (for example)

Reply:
Agreed, let's not do this in this PR but make a seperate issue and start cleaning multivuerse there (refactor)